js/editor-ta.js

```diff
@@ -0,0 +1,304 @@
+/**
+ * @constructor
+ * @param {DOM} elementId
+ * @param {Settings} settings
+ */
+function EditorTextArea(editorElement, settings) {
+  this.element_ = editorElement;
+  this.settings_ = settings;
+  this.numLines_ = 1;
+
+  this.setTheme();
+
+  const initFontSize = this.settings_.get('fontsize') + 'px';
+
+  // The main text area the user will be interacting with
+  this.textarea_ = document.createElement('textarea');
+  this.textarea_.setAttribute('id', 'editor-ta');
+  this.textarea_.style.fontSize = initFontSize;
+  this.textarea_.addEventListener('input', function() {
```

Comment:
I was avoiding arrow functions since the rest of the app was written when they didn't exist and i thought it would be a good idea to maintain the general style of the codebase. What do you think?

Reply:
Will keep this as is to match the current style of the app, introducing a mix of syntax is imo not worth the benefits that a arrow function gives

Reply:
sorry I didn't see this before! when I started working on text app I updated it to be able to work with ES6 and started using ES6 on all my changes, including arrow functions, const/let instead of ver, etc. so there is already a mix of syntax in the code base. i think all new code/changes should stick to ES6 (and ideally the old stuff updated, but not as a priority)

Reply:
Fair point, will update to arrow :)